action.yml

```diff
@@ -0,0 +1,15 @@
+name: Private Action Loader
+description: 'Allows a private action to be easily reused across repos'
+inputs:
+  repo-token:
+    description: 'Access token with read access to the repo containing action'
+    required: true
+  repo-name:
+    description: 'The organization/user and repo where action is stored'
+    required: true
```

Comment:
Wonder if this should also support the `user/repo@ref`, so versions can be pinned?

Reply:
Sounds easy enough.  I'll add an additional optional input for ref, similar to repo-branch.

Reply:
Added optional repo-sha input and updated readme